### PR TITLE
CY-1884 get all managers ips

### PR DESCRIPTION
**Pull request by YoniItzhak**

### CODE REVIEW

File: status-reporter/status_reporter/status_reporter.py [hunk 1]
```diff
@@ -105,23 +104,36 @@ def _build_report(self, status):
         return {'reporting_freq': self._current_reporting_freq,
                 'report': status}
 
-    def _update_managers_ips_list(self):
+    def _update_managers_ips_list(self, manager_ip):
+        url = 'https://{}:443/api/v3.1/managers'.format(manager_ip)
```
Comment (tyacbovi) [inline status-reporter/status_reporter/status_reporter.py]: are you sure we need to specify the  port?

File: status-reporter/status_reporter/status_reporter.py [hunk 2]
```diff
@@ -105,6 +104,37 @@ def _build_report(self, status):
...
+
+    def _report_status(self, manager_ip, report):
+        requests.put(self._build_full_reporting_url(manager_ip),
+                     data=report,
+                     headers={'tenant': 'default_tenant'},
+                     verify=self._ca_path,
+                     auth=(self._reporter_user_name,
+                           self._reporter_token))
+
+    def _report_and_update_managers_ips_list(self, report):
```
Comment (tyacbovi) [inline status-reporter/status_reporter/status_reporter.py]: I think this encapsulation is redundant

File: status-reporter/status_reporter/status_reporter.py [hunk 3]
```diff
@@ -119,19 +149,8 @@ def _report(self):
...
-                             headers={'tenant': 'default_tenant'},
-                             verify=self._ca_path,
-                             auth=(self._reporter_user_name,
-                                   self._reporter_token))
-                return
-            except Exception as e:
-                logger.debug('Error had occurred while trying to report '
-                             'status: '.format(e))
+        self._report_and_update_managers_ips_list(report)
+        self._report_and_update_managers_ips_list(report)
```
Comment (tyacbovi) [inline status-reporter/status_reporter/status_reporter.py]: there is duplication

File: status-reporter/status_reporter/status_reporter.py (hunk 1 above)
Comment (YoniItzhak) [reply status-reporter/status_reporter/status_reporter.py]: I'm not sure, but I think we'll change it anyway once we write the API, right?

File: status-reporter/status_reporter/status_reporter.py [hunk 4]
```diff
@@ -105,23 +104,36 @@ def _build_report(self, status):
...
+    def _update_managers_ips_list(self, manager_ip):
+        url = 'https://{}:443/api/v3.1/managers'.format(manager_ip)
+        headers = {'Tenant': 'default_tenant',
+                   'Content-type': 'application/json'}
+        response = requests.get(url,
+                                headers=headers,
+                                verify=self._ca_path,
+                                auth=(self._reporter_user_name,
+                                      self._reporter_token))
+        self._managers_ips = [manager.get('public_ip') for manager in
```
[the remark this answers is not in the document]
Comment (YoniItzhak) [reply status-reporter/status_reporter/status_reporter.py]: I'm not sure I understand this comment.

**Pull request merged**